discovery-provider/src/__init__.py

```diff
@@ -312,6 +315,10 @@ def configure_celery(flask_app, celery, test_config=None):
             "update_materialized_views": {
                 "task": "update_materialized_views",
                 "schedule": timedelta(seconds=60)
+            },
+            "update_network_peers": {
+                "task": "update_network_peers",
+                "schedule": timedelta(seconds=2)
```

Comment:
make this 15 or 20s? @dmanjunath @SidSethi whats a reasonable value. it def can't stay 2

Reply:
i think 30s is reasonable